lib/Controller/ChatController.php

```diff
@@ -146,11 +149,27 @@ public function sendMessage(string $message, string $actorDisplayName = ''): Dat
 			return new DataResponse([], Http::STATUS_NOT_FOUND);
 		}
 
+		$parent = null;
+		if ($replyTo !== 0) {
+			try {
+				$parent = $this->chatManager->getParentComment($this->room, (string) $replyTo);
+			} catch (NotFoundException $e) {
+				// Someone is trying to reply cross-rooms or to a non-existing message
+				return new DataResponse([], Http::STATUS_BAD_REQUEST);
+			}
+
+			$parentMessage = $this->messageParser->createMessage($this->room, $this->participant, $parent, $this->l);
+			$this->messageParser->parseMessage($parentMessage);
+			if ($parentMessage->getMessageType() === 'system' || $parentMessage->getMessageType() === 'command') {
```

Comment:
I guess that we are not adding any other special type of message anytime soon, but maybe explicitly limit replies only to _pure_ messages for now by checking if the type is not `comment` instead of checking if it is `system` or `command`?

Reply:
`¯\_(ツ)_/¯`

Reply:
the question is if a follow up type would be more likely to be repliable or not. If it is something like stickers or so, it could be repliable. If it is reall bots instead of commands, maybe not. but who knows.